scenes/abstract.py

```diff
@@ -1,19 +1,21 @@
 from abc import ABC
 from typing import Tuple
 
+import pymunk
 from pygame.event import Event
 from pygame.surface import Surface
 
 
 class AbstractScene(ABC):
     def __init__(self, display: Surface) -> None:
-        self.display = display
-
+        self.display: Surface = display
+        self.size_sc: tuple = display.get_size()
+        self.space: pymunk.Space() = pymunk.Space()
     def handle_events(self, events: Tuple[Event]) -> None:
         raise NotImplementedError()
 
-    def update(self):
-        raise NotImplementedError()
+    def update(self, step: int):
+        self.space.step(step)
```

Comment:
Absctract scene should not implement any of these methods. They must be implemented only in the concrete scene.